com.sap.cloud.lm.sl.mta/src/main/java/com/sap/cloud/lm/sl/mta/util/MetadataUtil.java

```diff
@@ -0,0 +1,18 @@
+package com.sap.cloud.lm.sl.mta.util;
+
+import java.util.Collections;
+import java.util.Map;
+
+import org.apache.commons.lang3.ObjectUtils;
+
+import com.sap.cloud.lm.sl.mta.model.Metadata;
+
+public class MetadataUtil {
+    public static Metadata getMetadataOrDefault(Metadata metadata) {
+        if (metadata != null) {
+            return metadata;
+        }
+        return new Metadata(ObjectUtils.defaultIfNull(Collections.<String, Map<String, Object>> emptyMap(),
```

Comment:
Can we move this default object to a static variable in the Metadata class? Then we'd be able to use the ObjectUtils.defaultIfNull method in the builders to retrieve it.